src/firebase-init.ts

```diff
@@ -0,0 +1,18 @@
+import { initializeApp } from "firebase/app";
+import { getAuth } from "firebase/auth";
+import { getFirestore } from "firebase/firestore";
+
+const firebaseConfig = {
+  projectId: "mapeo-webmaps-staging",
+  appId: "1:354071501370:web:a92ea6497d55c4dd9ab303",
+  storageBucket: "mapeo-webmaps-staging.appspot.com",
+  apiKey: "AIzaSyAddUwtJxCTq3VImtID0S-5beOHiJtKTe4",
+  authDomain: "mapeo-webmaps-staging.firebaseapp.com",
+  messagingSenderId: "354071501370",
+  measurementId: "G-GLMDFRWPJD",
+}
+
+export const firebaseApp = initializeApp(firebaseConfig)
+
+export const db = getFirestore(firebaseApp)
+export const auth = getAuth();
```

Comment:
How do we typically handle environment variables for firebase stuff?

In the previous version we were [calling an init function](https://github.com/digidem/mapeo-webmaps/blob/71822426d076be136170ef05326162966aae514d/src/index.js#L33) that allowed us to require in the config in production (as far as i can tell this is some firebase-specific magic?)

In firebase 9 we need to be able to export `firebaseApp`, `db` and `auth` so it makes that a little weirder. Could call a `getConfig` function for example then run things like we do here?